pandas/core/indexers.py

```diff
@@ -65,29 +69,24 @@ def is_empty_indexer(indexer, arr_value: np.ndarray) -> bool:
 def check_setitem_lengths(indexer, value, values) -> None:
     """
     Validate that value and indexer are the same length.
-
-    An special-case is allowed for when the indexer is a boolean array
-    and the number of true values equals the length of ``value``. In
-    this case, no exception is raised.
+    A special-case is allowed for when the indexer is a
+    boolean array, and the number of true values, equals to
+    the length of ``value``.
```

Comment:
The previous sentence is still correcter english I think (or at least the comma after "values" is not in a correct place)

Reply:
@jorisvandenbossche I'm sorry, I don't understand what should I fix here? can you be more specific?

Reply:
There is still a diff show here, and the original version is a correcter sentence I think. The comma (",") in "the number of true values, equals .." is not correct

Reply:
I see it now, you're 100% correct.